internal/server/cache/svg.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cache
```

Comment:
instead of doing this, why not using the protobuf when writing the cache into sql? I really think we should try to not duplicate logic across repo.

Reply:
That SG. How should we reference the protos in the import repo?

Reply:
maybe copy the proto definition for now and have a todo to use submodule repo to share the proto across?

Reply:
ok.